protos/dcs/mission/v0/mission.proto

```diff
@@ -538,11 +538,14 @@ message StreamUnitsResponse {
   }
 
   oneof update {
-    // The unit is either new or its position changed.
-    dcs.common.v0.Unit unit = 1;
+    // The unit is new.
+    dcs.common.v0.Unit new = 1;
+
+    // The unit's state changed (position, orientation, ...).
+    dcs.common.v0.Transform changed = 2;
```

Comment:
By grouping everything related to 2D/3D position/transform into one message, we could easily distinguish in the units stream between new and changed, and skip re-sending static unit information again for the latter. Just in case this is something we want to do - I don't have a strong oppinion here myself, so I'd fine with keeping it as is.

Reply:
I would much prefer we do not do this. Doing so forces clients to maintain state now which not all clients will want to do. I would prefer we send the complete set of unit information, including static information, each time. I think the performance impact of this would be negligable.